src/lib/react/React.hx

```diff
@@ -51,7 +55,9 @@ extern class React
 		If you are using an earlier release of React, use callback refs instead
 		https://reactjs.org/docs/refs-and-the-dom.html#callback-refs
 	**/
+	#if react_ref_api
```

Comment:
I think Haxe uses the first doc comment it comes across, so if you don't include the doc comment in the conditionals, the following field will have the wrong one.

![](https://i.imgur.com/0AkKrPP.png)

Reply:
Thanks for the tip